```python-traceback
02:08 >       assertThrowRegex(
02:08             'Failed to set file mode on remote temporary file',
02:08             execute=True)
02:08 
02:08 test/units/plugins/action/test_action.py:429: 
02:08 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
02:08 test/units/plugins/action/test_action.py:351: in assertThrowRegex
02:08     self.assertRaisesRegex(
02:08 E   AssertionError: "Failed to set file mode on remote temporary file" does not match "Failed to set file mode or acl on remote temporary files (rc: 1, err: )"
```

--- a/test/units/plugins/action/test_action.py
+++ b/test/units/plugins/action/test_action.py
@@ -427,7 +427,7 @@ class TestActionBase(unittest.TestCase):
             'stderr': '',
         }
         assertThrowRegex(
-            'Failed to set file mode on remote temporary file',
+            'Failed to set file mode or acl on remote temporary files',
             execute=True)
         action_base._remote_chmod.return_value = {
             'rc': 0,

